plugin-build/src/main/kotlin/io/sentry/android/gradle/util/SentryAndroidSdkChecker.kt

```diff
@@ -18,35 +19,46 @@ fun Project.detectSentryAndroidSdk(
         logger.warn {
             "Unable to find configuration $configurationName for variant $variantName."
         }
-        sentryModulesService.get().modules = emptyMap()
+        sentryModulesService.get().sentryModules = emptyMap()
+        sentryModulesService.get().externalModules = emptyMap()
         return
     }
 
     configProvider.configure { configuration ->
         configuration.incoming.afterResolve {
-            val sentryModules = it.resolutionResult.allComponents.filterSentryModules(logger)
+            val allModules = it.resolutionResult.allComponents.versionMap(logger)
+            val sentryModules = allModules.filter { (identifier, _) ->
+                identifier.group == "io.sentry"
+            }.toMap()
+
+            val externalModules = allModules.filter { (identifier, _) ->
```

Comment:
`m`: Maybe worth adding a quick integration test, that adds `androidx.sqlite:sqlite-framework` to the dependencies of a fixture, and then check that it's in the `externalModules` of the `SentryModulesService`? Similar to what we do for testing that sentry modules are there (`SentryAndroidSdkCheckerTest`)